app/helpers/cases_helper.rb

```diff
@@ -328,4 +328,15 @@ def show_escalation_deadline?(kase)
       kase.has_attribute?(:escalation_deadline) &&
       kase.within_escalation_deadline?
   end
+
+  def get_sar_recipient_label(recipient)
+    if recipient == 'requester_recipient'
+      t('helpers.label.offender_sar.recipient_type.recipient')
+    elsif recipient == 'subject_recipient'
+      t('helpers.label.offender_sar.recipient_type.data_subject')
+    else
+      t('helpers.label.offender_sar.recipient_type.third_party')
+    end
+  end
+
```

Comment:
Could we move this into the offender_decorator, maybe? 

Also, wondering if we can get rid of this conditional by using the same approach as we did in the offender_decorator for `subject_type_display` if we wire up the keys in the en.yml file using the same strings as the values for recipient here. 

Compare with 
![image](https://user-images.githubusercontent.com/1161161/88113770-6786fc80-cbaa-11ea-8d24-b23dbed418df.png)

So we'd have keys in the yaml file like `helpers.label.offender_sar.recipient.requester_recipient` or more simply `helpers.label.offender_sar.requester_recipient` and then the method becomes something like 

```suggestion
def get_sar_recipient_label(recipient)
  I18n.t('helpers.label.offender_sar.recipient.' + recipient)
end
```
(... but using the equivalent form for a decorator i.e. `object.recipient`)